openlibrary/templates/books/edition-sort.html

```diff
@@ -60,61 +54,58 @@
                 $if book.languages:
                     in $', '.join(l.name for l in book.languages)
             $if book.edition_name:
-                -
-                $book.edition_name
+                - $book.edition_name
             </div>
         </div>
     </td>
 
-    $if 'printdisabled' in collection or 'lendinglibrary' in collection:
-        <td class="icon read">
-        <div class="bbbb"></div>
+    $ availability_class = 'zzzz' if (not availability or availability.get('status') == 'error') else 'aaaa' if (availability.get('status') == 'open') else 'bbbb' if (availability.get('status') == 'borrow_available') else 'cccc' if (availability.get('status') == 'borrow_unavailable') else 'dddd'
```

Comment:
QUESTION: What's the significance of these strange class names? e.g. `zzzz`, `aaaa`, etc.

Reply:
legacy, it's used as an alphabetic sort mechanism. There's probably a better way to do it, but I didn't want to re-invent the sort strategy if it wasn't broken.